samples/Basic-TFRecorder-Usage.ipynb

```diff
@@ -139,75 +149,20 @@
-       "        if (typeof window.interactive_beam_jquery == 'undefined') {\n",
-       "          var jqueryScript = document.createElement('script');\n",
-       "          jqueryScript.src = 'https://code.jquery.com/jquery-3.4.1.slim.min.js';\n",
-       "          jqueryScript.type = 'text/javascript';\n",
-       "          jqueryScript.onload = function() {\n",
-       "            var datatableScript = document.createElement('script');\n",
-       "            datatableScript.src = 'https://cdn.datatables.net/1.10.20/js/jquery.dataTables.min.js';\n",
-       "            datatableScript.type = 'text/javascript';\n",
-       "            datatableScript.onload = function() {\n",
-       "              window.interactive_beam_jquery = jQuery.noConflict(true);\n",
-       "              window.interactive_beam_jquery(document).ready(function($){\n",
-       "                \n",
-       "              });\n",
-       "            }\n",
-       "            document.head.appendChild(datatableScript);\n",
-       "          };\n",
-       "          document.head.appendChild(jqueryScript);\n",
-       "        } else {\n",
-       "          window.interactive_beam_jquery(document).ready(function($){\n",
-       "            \n",
-       "          });\n",
-       "        }"
-      ]
-     },
-     "metadata": {},
-     "output_type": "display_data"
-    },
-    {
-     "data": {
-      "application/javascript": [
-       "\n",
-       "        var import_html = () => {\n",
-       "          ['https://raw.githubusercontent.com/PAIR-code/facets/1.0.0/facets-dist/facets-jupyter.html'].forEach(href => {\n",
-       "            var link = document.createElement('link');\n",
-       "            link.rel = 'import'\n",
-       "            link.href = href;\n",
-       "            document.head.appendChild(link);\n",
-       "          });\n",
-       "        }\n",
-       "        if ('import' in document.createElement('link')) {\n",
-       "          import_html();\n",
-       "        } else {\n",
-       "          var webcomponentScript = document.createElement('script');\n",
-       "          webcomponentScript.src = 'https://cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/1.3.3/webcomponents-lite.js';\n",
-       "          webcomponentScript.type = 'text/javascript';\n",
-       "          webcomponentScript.onload = function(){\n",
-       "            import_html();\n",
-       "          };\n",
-       "          document.head.appendChild(webcomponentScript);\n",
-       "        }"
-      ]
-     },
-     "metadata": {},
-     "output_type": "display_data"
-    },
-    {
-     "data": {
-      "text/plain": [
-       "{'job_id': 'DirectRunner',\n",
-       " 'metrics': {'rows': 6, 'good_images': None, 'bad_images': 6},\n",
-       " 'tfrecord_dir': './out/tfrecorder-20201028-160301-to-tfr'}"
-      ]
-     },
-     "execution_count": 4,
-     "metadata": {},
-     "output_type": "execute_result"
+     "ename": "FileNotFoundError",
```

Comment:
I'd suggest clearing the outputs of the notebooks before you check them in to Git.

Reply:
All done and have pushed the changes